### DISCUSSION TITLE
Running Local Bot API Server, when trying to connect the bot, I get Not Implemented error on the server

### DISCUSSION THREAD

**Discussion by iargaman** [python-telegram-bot/python-telegram-bot#3557] [Q&A] [answered]
[HttpReader.cpp:722][!HttpInboundConnection]	Not Implemented [type:PRI][head:PRI * HTTP/2.0
[HttpConnectionBase.cpp:134][!HttpInboundConnection]	[Error : 501 : Not Implemented]
![image](https://user-images.githubusercontent.com/3229399/219855175-34d03584-12cb-49d8-9506-d6993786a0d2.png)

I'm running on port 9995
when trying to access getMe via the browser, I see the bot details
localhost:9995/bot<MY TOKEN>/getMe

but from the code:

    BASE_URL = "http://localhost:9995/bot"
    BASE_FILE_URL = "http://localhost:9995/file/bot"
    builder = Application.builder()
    builder.token(TOKEN)
    builder.local_mode(local_mode=True)
    builder.base_url(BASE_URL)
    builder.base_file_url(BASE_FILE_URL)
    application = builder.build()
    application.run_polling()

then I receive the above error on my server 

I checked the HttpReader.cpp file in line :722
![image](https://user-images.githubusercontent.com/3229399/219855256-ce7836f3-46c8-4401-afe8-7cf04c38b3d1.png)
it seems the bot trying to send "PRI" type that is not supported 

any suggestions?
tried almost everything

**Comment by harshil21** [comment, marked as answer]
It is a known issue that we cannot use the local Bot API server with HTTP/2 (#3556)

As a workaround, add .http_version('1.1') and .get_updates_http_version('1.1') to your AppBuilder

**Reply by iargaman** [reply]
great, it seems to pass this issue, getting now another problem but i will try investigate first
Thank you
